e2e/nomostest/gitproviders/local.go

```diff
@@ -75,3 +68,8 @@ func (l *LocalProvider) DeleteRepositories(...string) error {
 func (l *LocalProvider) DeleteObsoleteRepos() error {
 	return nil
 }
+
+// WithPortForwarder sets the PortForwarder for the in-cluster git server
+func (l *LocalProvider) WithPortForwarder(pf *portforwarder.PortForwarder) {
```

Comment:
With- is usually used for options and builders. This is just a setter. But we usually avoid setters in Go and just make the field public.

Reply:
The general rule is that all public fields need to be set by the caller who constructs the struct instance, or at least before other method use.

Reply:
done